### PR TITLE
Create testing bundle (#23)

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- Create a testing bundle which contains plans with minimal and full configuration

### CODE REVIEW

File: bundles/testing-0.0.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+tags: test
+labels:
+  showcase: "true"
+providerDisplayName: SAP
+longDescription: "Testing bundle which contains dummy ConfigMap"
+documentationURL: https://todo.com/
+supportURL: https://todo.com/
+imageURL: https://todo.com/
+bindable: true
```
Comment (polskikiel) [inline bundles/testing-0.0.1/meta.yaml]: Please, add the following properties to this file: 
```
planUpdatable: false
bindingRetrievable: false
```

File: bundles/testing-0.0.1/plans/full/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: full
+id: a6078799-70a1-4674-af91-aba44dd6a56
+description: "Full plan with all possible settings"
+displayName: Full
+bindable: true
```
Comment (polskikiel) [inline bundles/testing-0.0.1/plans/full/meta.yaml]: Please, add the following property to this file: 
```
free: true
```

File: bundles/testing-0.0.1/plans/minimal/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+name: minimal
+id: 631dae68-98e1-4e45-b79f-1036ca5b29cb
+description: "Minimal plan"
+displayName: Micro
```
Comment (polskikiel) [inline bundles/testing-0.0.1/plans/minimal/meta.yaml]: Please, add the following property to this file: 
```
free: true
```

File: bundles/testing-0.0.1/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+The Testing bundle is a simple bundle which contains one resource - a ConfigMap and a Deployment. It shows all possible parameters.
+
+## Details
+
+The bundle contains two plans:
+- minimal - contains only necessary parameters, be aware there is no binding template defined.
+- full - contains all possible parameters
+
+It is a show case how to define chart with minimal and full configuration.
```
Comment (klaudiagrz) [inline bundles/testing-0.0.1/README.md]:
```suggestion
This bundle is a showcase on how to define a chart with minimal and full configuration.
```

File: bundles/testing-0.0.1/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+```
+
+## Overview
+
+The Testing bundle is a simple bundle which contains one resource - a ConfigMap and a Deployment. It shows all possible parameters.
+
+## Details
+
+The bundle contains two plans:
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply bundles/testing-0.0.1/README.md]: The bundle contains the following plans

**Pull request merged**